### PR TITLE
chore: update hub-auth rev for external references to hub-auth test/bdd

### PR DESCRIPTION
**Pull request by Baha-sk**
Signed-off-by: Baha Shaaban <baha.shaaban@securekey.com>

### CODE REVIEW

File: test/bdd/go.mod [hunk 1]
```diff
@@ -16,7 +16,7 @@ require (
 	github.com/pquerna/cachecontrol v0.0.0-20200921180117-858c6e7e6b7e // indirect
 	github.com/tidwall/gjson v1.6.7
 	github.com/trustbloc/edge-core v0.1.7-0.20210310142750-7eb11997c4a9
-	github.com/trustbloc/hub-auth v0.0.0-00010101000000-000000000000
```
Comment (DRK3) [inline test/bdd/go.mod:19]: This should still work... what error are you getting?

**Review by DRK3 — CHANGES_REQUESTED** [review]

File: test/bdd/go.mod (hunk 1 above)
Comment (Baha-sk) [reply test/bdd/go.mod:19]: this works here fine, but adding hub-auth/test/bdd revision in the kms project fails to resolve  revision `000000000000` apparently @DRK3 @aholovko at this line:
https://github.com/trustbloc/kms/blob/main/test/bdd/go.mod#L21

Comment (Baha-sk) [reply test/bdd/go.mod:19]: go mod tidy in `kms/test/bdd/` gives this error when updating to `v0.0.0-20210310162148-949bbe290351`:
```
go: github.com/trustbloc/hub-auth/test/bdd@v0.0.0-20210310162148-949bbe290351 requires
	github.com/trustbloc/hub-auth@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
```
and if providing the full hub-auth revision (`v0.1.7-0.20210310162148-949bbe290351`), it gives this error:
```
go: github.com/trustbloc/hub-auth/test/bdd@v0.1.7-0.20210310162148-949bbe290351: invalid pseudo-version: preceding tag (v0.1.6) not found
```

**Review by DRK3 — APPROVED** [review]

File: test/bdd/go.mod (hunk 1 above)
Comment (DRK3) [reply test/bdd/go.mod:19]: As discussed: @aholovko Will in a future commit update `kms` to not import from the bdd test

**Review by aholovko — APPROVED** [review]

File: test/bdd/go.mod (hunk 1 above)
Comment (Baha-sk) [reply test/bdd/go.mod:19]: yes added a // TODO above this line to be removed in a future change

**Pull request merged**